lib/src/draw_config_context.dart

```diff
@@ -53,15 +53,23 @@ final kNotSet = null;
 /// The [DRAWConfigContext] class provides an interface to store.
 /// Load the DRAW's configuration file draw.ini.
 class DRAWConfigContext {
-  static final Uri kDefaultAccessToken =
-      Uri.parse(r'https://www.reddit.com/api/v1/access_token');
-  static final Uri kDefaultAuthorizeUri =
-      Uri.parse(r'https://reddit.com/api/v1/authorize');
+  static final String kDefaultAccessToken =
```

Comment:
`static const` here and below. Also, please remove duplicate fields from the global space.